app/routes/bundesIdent/survey/dropout.tsx

```diff
@@ -0,0 +1,114 @@
+import {
+  ActionFunction,
+  LoaderFunction,
+  json,
+  redirect,
+} from "@remix-run/node";
+import invariant from "tiny-invariant";
+import { Form, useNavigation } from "@remix-run/react";
+import { isEmpty } from "lodash";
+import {
+  BreadcrumbNavigation,
+  Button,
+  ButtonContainer,
+  ContentContainer,
+  Headline,
+  Textarea,
+} from "~/components";
+import { authenticator } from "~/auth.server";
+import { createSurvey } from "~/domain/survey";
+import { commitSession, getSession } from "~/session.server";
+
+const SURVEY = "survey";
+const SURVEY_CATEGORY = "dropout";
+const IDENT_OPTION_PATH = "/identifikation";
+
+export const loader: LoaderFunction = async ({ request }) => {
+  const sessionUser = await authenticator.isAuthenticated(request, {
+    failureRedirect: "/anmelden",
+  });
+
+  const session = await getSession(request.headers.get("Cookie"));
+  const hasSurveyShown = session.get("hasSurveyShown") || true;
+  session.set("hasSurveyShown", hasSurveyShown);
```

Comment:
@pgurusinga nit-pick: `session.set("hasSurveyShown", true);` might be a bit clearer to the reader.